# Date and time message as reported by vehicle.

std_msgs/Header header

int32 year    # Year in the range [2000,2255]
uint8 month   # Month in the range [1,12]
uint8 day     # Day in the range [1,31]
uint8 hour    # Hour in the range [0,23] 
uint8 minute  # Minute in the range [0,60]
uint8 second  # Second in the range [0,60]